### PR TITLE
add logger to schedulers

### LINKED ISSUE #50: add logger to scheduler

### PR DESCRIPTION
**Pull request by cappyzawa**
## What is this?
* add the logger to schedulers
* remove no using files/struct
* fix #50

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -33,10 +33,6 @@ func (a *API) run(port string) error {
 	return http.ListenAndServe(":"+port, mux)
 }
 
-type remindTimer struct {
```
Comment (cappyzawa) [inline main.go:36]: this struct is not used

File: scheduler/reminder.go [hunk 2]
```diff
@@ -40,47 +42,86 @@ func (r *Reminder) calculateRemainTime(timeStr string) (time.Duration, string, e
...
+				zap.Duration("remain", remain),
+				zap.String("next remind", nextHour),
+			)
 		}
 	}
 
 }
 
-func (r *Reminder) remind(id string) error {
+func (r *Reminder) Remind(id string) error {
```
Comment (cappyzawa) [inline scheduler/reminder.go:87]: this function is exported to enable to test

File: scheduler/reminder.go [hunk 3]
```diff
@@ -40,47 +42,86 @@ func (r *Reminder) calculateRemainTime(timeStr string) (time.Duration, string, e
...
 			remain, nextHour, err = r.calculateRemainTime(nextHour)
 			if err != nil {
+				r.Logger.Error("failed to calculate remain time",
+					zap.Error(err),
+				)
 				return err
 			}
+			r.Logger.Info("before reminding",
+				zap.Time("now", nowJST),
+				zap.Float64("remain", remain.Hours()),
```
Comment (mr-oliva) [inline scheduler/reminder.go:79]: Float??

**Review by mr-oliva — APPROVED** [review]
👏

**Pull request merged**